src/_gettsim/parameters/kinderzuschl.yaml

```diff
@@ -226,8 +246,8 @@ existenzminimum:
       paare: 1008
       kinder: 180
     reference: >-
-      10. Existenzminimumsbericht, Bundestag Drucksache 17/5550, Übersicht 4. Werte für
-      Paare unverändert.
+      10. Existenzminimumsbericht, Bundestag Drucksache 17/5550,
```

Comment:
Is there any auto-formatting going on?

Reply:
The pre-commit hooks strangely kept complaining about the length of a few lines in the .yaml-files, even though I hadn't changed them, so I had to adjust them so that the hooks would run through